src/languages/en.ts

```diff
@@ -1897,7 +1897,7 @@ export default {
             categoryRequiredError: 'Category name is required.',
             existingCategoryError: 'A category with this name already exists.',
             invalidCategoryName: 'Invalid category name.',
-            importedFromAccountingSoftware: 'The categories below are imported from your',
```

Comment:
NAB. Use single quote instead of back quote

(same for Spanish)

Reply:
fixed: https://github.com/Expensify/App/pull/40496/commits/5235160a2799870585985a2d5ad9a78a23fadba7